AsyncGenerator.yml

```diff
@@ -28,6 +28,65 @@
   registerPlugin:
   - type: AsyncGenerator.Core.Plugins.EmptyRegionRemover
     assemblyName: AsyncGenerator.Core
+- filePath: StackExRedis\NHibernate.Caches.StackExRedis\NHibernate.Caches.StackExRedis.csproj
+  targetFramework: net461
+  concurrentRun: true
+  applyChanges: true
+  analyzation:
+    methodConversion:
+    - conversion: Ignore
+      hasAttributeName: ObsoleteAttribute
+    - conversion: Smart
+      containingTypeName: AbstractRegionStrategy
+    - conversion: Smart
+      containingTypeName: RedisKeyLocker
+    callForwarding: true
+    cancellationTokens:
+      guards: true
+      methodParameter:
+      - parameter: Required
+      requiresCancellationToken:
+      - containingType: NHibernate.Caches.StackExRedis.RedisCache
+        name: GetMany
+      - containingType: NHibernate.Caches.StackExRedis.RedisCache
+        name: PutMany
+      - containingType: NHibernate.Caches.StackExRedis.RedisCache
+        name: RemoveMany
+      - containingType: NHibernate.Caches.StackExRedis.AbstractRegionStrategy
+        name: Get
+      - containingType: NHibernate.Caches.StackExRedis.AbstractRegionStrategy
+        name: GetMany
+      - containingType: NHibernate.Caches.StackExRedis.AbstractRegionStrategy
+        name: Put
+      - containingType: NHibernate.Caches.StackExRedis.AbstractRegionStrategy
+        name: PutMany
+      - containingType: NHibernate.Caches.StackExRedis.AbstractRegionStrategy
+        name: Remove
+      - containingType: NHibernate.Caches.StackExRedis.AbstractRegionStrategy
+        name: RemoveMany
+      - containingType: NHibernate.Caches.StackExRedis.AbstractRegionStrategy
+        name: Unlock
+      - containingType: NHibernate.Caches.StackExRedis.AbstractRegionStrategy
+        name: UnlockMany
+      - containingType: NHibernate.Caches.StackExRedis.AbstractRegionStrategy
+        name: Clear
+      - containingType: NHibernate.Caches.StackExRedis.RedisKeyLocker
+        name: Unlock
+      - containingType: NHibernate.Caches.StackExRedis.RedisKeyLocker
+        name: UnlockMany
+    scanMethodBody: true
+    searchAsyncCounterpartsInInheritedTypes: true
```

Comment:
This property requires AsyncGenerator 0.13, while 0.12 was used. So I have updated it.